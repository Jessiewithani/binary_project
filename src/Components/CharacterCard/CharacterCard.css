.card {
    background-color: rgba(0, 0, 0, 0.4);
    border-radius: 20px;
    box-shadow: 7px 8px 5px 0px rgba(0,0,0,0.51);
    color: #B7E4F9FF;
    height: 620px;
    margin: auto;
    margin-bottom: 20px;
    margin-top:20px;
    max-width: 300px;
}

img {
    border-radius: 20px 20px 0px 0px;
    width: 100%;
}

.card-header {
    position: relative;
}
.character-name {
    background-color: rgba(0, 0, 0, 0.3);
    bottom: 0px;
    color: gainsboro;
    position: absolute;
    width: 100%;

}

.character-info-container {
    height: 100%;
    padding: 0 20px;
}

.status-info {
    align-items: center;
    display: flex;
    justify-content: space-between;
}

.info {
    align-items: center;
    border-top: 1px solid white;
    display: flex;
    justify-content: space-between;
}