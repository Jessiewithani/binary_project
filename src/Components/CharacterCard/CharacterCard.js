import React from 'react';
import '../CharacterCard/CharacterCard.css'

const CharacterCard = ({ id, name, status, species, type, gender, location, image}) => {
    // console.log(location);
    return(
        <section className="card">
            <div className="card-header">
                <div className="character-image">
                    <img src={image} alt="" />
                </div>
                <div className="character-name">
                    <h2>{name}</h2>
                </div>
            </div>
            <div className="character-info-container">
                <div className="status-info">
                    <span>status:</span><h4>{status}</h4>
                </div>
                <div className="info">
                    <span>species:</span><h4>{species}</h4>
                </div>
                <div className="info">
                    <span>type:</span><h4>{type}</h4>
                </div>
                <div className="info">
                    <span>gender:</span><h4>{gender}</h4>
                </div>
                <div className="info">
                    <span>location:</span><h4>{location.name}</h4>
                </div>
            </div>
            {/* <h3>{image}</h3> */}
        </section>
    )
}

export default CharacterCard;