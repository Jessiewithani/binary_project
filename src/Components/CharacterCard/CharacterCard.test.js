import React from 'react';
import CharacterCard from '../CharacterCard/CharacterCard';

describe('CharacterCard', () => {
    it.skip('should...', () => {
  
    })
  })
  