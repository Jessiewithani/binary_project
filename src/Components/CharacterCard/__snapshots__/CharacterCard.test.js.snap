// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CharacterCard should match the snapshot with all the correct data passed in 1`] = `ShallowWrapper {}`;
